assets/src/styles/editorStyle.scss

```diff
@@ -26,6 +26,7 @@
 @import "components/Preview";
 @import "components/CharacterCounter";
 @import "campaigns/base/fonts";
+@import "campaigns/base/google_fonts";
```

Comment:
Should we add the same to [`assets/src/styles/campaigns.scss`](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/f918ee7627cfe4329b202d5ec33d1c122ccfe887/assets/src/styles/campaigns.scss#L11) ?